src/server/controllers/post.controller.ts

```diff
@@ -179,6 +180,26 @@ export const updatePostHandler = async ({
         throw throwBadRequestError('You must select a tag for this collection');
       }
 
+      if (collection.metadata.entriesRequireTitle && !post.title) {
+        throw throwBadRequestError('Your entry must have a title to be submitted');
+      }
+
+      if (collection.metadata.entriesRequireTools) {
+        // Check all images within this post has tools:
+        const exists = await dbWrite.image.findFirst({
```

Comment:
Should we use dbRead instead?
```suggestion
        const exists = await dbRead.image.findFirst({
```